numba/tests/test_array_methods.py

```diff
@@ -187,6 +187,10 @@ def np_unique(a):
     return np.unique(a)
 
 
+def np_repeat(a, repeats, axis):
+    return a.repeat(repeats, axis)
```

Comment:
This function is using `ndarray.repeat`, i.e. the `repeat` method on the `ndarray` class. However, the implementation here:
https://github.com/numba/numba/pull/3519/files#diff-81d8f85ead1fb3ae948c94ed8d2a969eR1841
is overloading `numpy.repeat`, this is the reason for the error:
```
Unknown attribute 'repeat' of type array(int64, 0d, C)
```
To start out, perhaps just work on getting `numpy.repeat` to work (i.e. change this call to `np.repeat(a, repeats, axis)`?